config/deployer.sample.json

```diff
@@ -111,5 +111,13 @@
         "Query": "histogram_quantile(0.99, sum(rate(mattermost_api_time_bucket[1m])) by (le))"
       }
     ]
-  }
+  },
+  "StorageSize": {
```

Comment:
Should this be `StorageSizes`? Not a big deal, just a nit.

Reply:
Yeah, that makes more sense, thanks! Done in https://github.com/mattermost/mattermost-load-test-ng/pull/659/commits/b931707620510de25362c09c5a1b62f017362d28